.github/workflows/export-requirements.yml

```diff
@@ -33,6 +34,8 @@ jobs:
         run: poetry export --format requirements.txt --output requirements.txt
 
       - name: Commit changes if any
+        # Skip this step if being run by nektos/act
+        if: ${{ !env.ACT }}
```

Comment:
The environment variable `ACT` is always set when run by `act`.

The github event `github.event.act` _may_ be set if the developer remembered to pass it in.